src/main/java/bdStudents/App.java

```diff
@@ -0,0 +1,49 @@
+package bdStudents;
+
+import java.util.List;
+
+public class App {
+    public static void main(String[] args) {
+
+        StudentDAO studentDAO = new StudentDAO();
```

Comment:
create session factory here and inject it into studenDao via constructor parameter